src/Solnet.Examples/AssociatedTokenAccountsExample.cs

```diff
@@ -0,0 +1,157 @@
+using Solnet.Programs;
+using Solnet.Rpc;
+using Solnet.Rpc.Builders;
+using Solnet.Rpc.Core.Http;
+using Solnet.Rpc.Messages;
+using Solnet.Rpc.Models;
+using Solnet.Rpc.Types;
+using Solnet.Wallet;
+using System;
+using System.Threading;
+
+namespace Solnet.Examples
+{
+    public class AssociatedTokenAccountsExample : IExample
+    {
+        
+        private static readonly IRpcClient RpcClient = ClientFactory.GetClient(Cluster.TestNet);
+
+        private const string MnemonicWords =
+            "route clerk disease box emerge airport loud waste attitude film army tray " +
+            "forward deal onion eight catalog surface unit card window walnut wealth medal";
+
+        public void Run()
+        {
+            Wallet.Wallet wallet = new Wallet.Wallet(MnemonicWords);
+            
+            /*
+             * The following region creates and initializes a mint account, it also creates a token account
+             * that is initialized with the same mint account and then mints tokens to this newly created token account.
+             */
+            #region Create and Initialize a token Mint Account
+            
+            
+            RequestResult<ResponseValue<BlockHash>> blockHash = RpcClient.GetRecentBlockHash();
+            
+            ulong minBalanceForExemptionAcc =
+                RpcClient.GetMinimumBalanceForRentExemption(TokenProgram.TokenAccountDataSize).Result;
+            ulong minBalanceForExemptionMint =
+                RpcClient.GetMinimumBalanceForRentExemption(TokenProgram.MintAccountDataSize).Result;
+            
+            Console.WriteLine($"MinBalanceForRentExemption Account >> {minBalanceForExemptionAcc}");
+            Console.WriteLine($"MinBalanceForRentExemption Mint Account >> {minBalanceForExemptionMint}");
+
+            Account ownerAccount = wallet.GetAccount(10);
+            Account mintAccount = wallet.GetAccount(1002);
+            Account initialAccount = wallet.GetAccount(1102);

```

Comment:
I don't know what these magic numbers mean. (Although this is just a test)

Reply:
These numbers are the index for the account, i.e. When you get a mnemonic on Sollet and use "Add Account" in the menu it's equivalent to using an index like that and incrementing it for everytime you go on the menu to add a new account without importing by private key.